src/syft/worker/virtual/virtual_client.py

```diff
@@ -1,6 +1,8 @@
 from ..client import Client
+from typing import final
 
 
+@final
```

Comment:
@iamtrask @tudorcebere what do you think?
This says that this class should not be inherited anymore